pandas/core/indexes/multi.py

```diff
@@ -1046,16 +1046,17 @@ def _shallow_copy(
         result._cache.pop("levels", None)  # GH32669
         return result
 
-    def _shallow_copy_with_infer(self, values, **kwargs):
-        # On equal MultiIndexes the difference is empty.
+    def symmetric_difference(self, other, result_name=None, sort=None):
```

Comment:
did you really rename this?

Reply:
symmetric_difference is the only method for which the logic in MultiIndex._shallow_copy_with_infer was needed, so basically yes.